### PR TITLE
Update mgmtMonitorWorkbookDevOps.json

### PR DESCRIPTION
**Pull request by lanicolas**
Changed screenshots and updated workbook

### CODE REVIEW

File: azure_jumpstart_arcbox/artifacts/mgmtMonitorWorkbookDevOps.json [hunk 1]
```diff
@@ -446,7 +446,7 @@
                       "crossComponentResources": [
                         "{subscription}"
                       ],
-                      "value": "/subscriptions/<subscriptionId>/resourceGroups/workbook/providers/Microsoft.OperationalInsights/workspaces/<workspaceName>",
+                      "value": "/subscriptions/<subscriptionId>/resourceGroups/ArcBoxDevOps/providers/Microsoft.OperationalInsights/workspaces/<workspaceName>",
```
Comment (dkirby-ms) [inline azure_jumpstart_arcbox/artifacts/mgmtMonitorWorkbookDevOps.json:449]: this looks like a hardcoded RG name. is that intentional?

File: azure_jumpstart_arcbox/artifacts/mgmtMonitorWorkbookDevOps.json [hunk 2]
```diff
@@ -915,14 +914,11 @@
                   "aggregation": 5,
                   "showAnalytics": true,
                   "title": "Azure Arc enabled kubernetes - Pod Restart Trend",
```
Comment (dkirby-ms) [inline azure_jumpstart_arcbox/artifacts/mgmtMonitorWorkbookDevOps.json:916]:
```suggestion
                  "title": "Azure Arc-enabled Kubernetes - Pod Restart Trend",
```

File: azure_jumpstart_arcbox/artifacts/mgmtMonitorWorkbookDevOps.json [hunk 3]
```diff
@@ -941,14 +937,11 @@
                   "aggregation": 5,
                   "showAnalytics": true,
                   "title": "Azure Arc enabled kubernetes - Container restart trend",
```
Comment (dkirby-ms) [inline azure_jumpstart_arcbox/artifacts/mgmtMonitorWorkbookDevOps.json:939]:
```suggestion
                  "title": "Azure Arc-enabled Kubernetes - Container restart trend",
```

File: azure_jumpstart_arcbox/artifacts/mgmtMonitorWorkbookDevOps.json [hunk 4]
```diff
@@ -1037,14 +1030,11 @@
                   "aggregation": 5,
                   "showAnalytics": true,
                   "title": "Azure Arc enabled kubernetes - Container Status for Pods",
```
Comment (dkirby-ms) [inline azure_jumpstart_arcbox/artifacts/mgmtMonitorWorkbookDevOps.json:1032]:
```suggestion
                  "title": "Azure Arc-enabled Kubernetes - Container Status for Pods",
```

**Review by dkirby-ms — CHANGES_REQUESTED** [review]

**Comment by lanicolas** [comment]
fixed, sorry for the oversight

**Pull request closed without merging**